ansible/ocs.yaml

```diff
@@ -0,0 +1,292 @@
+---
+- hosts: convergence_base
+  gather_facts: false
+  become: true
+  user: root
+  vars_files: vars/default.yaml
+  roles:
+  - oc_local
```

Comment:
This is broken, except in the case that convergence_base is also localhost. The reason is that oc_local was only configured on localhost.